modin/backends/pandas/query_compiler.py

```diff
@@ -1860,3 +1860,78 @@ def iloc_mut(partition, row_internal_indices, col_internal_indices, item):
             item_to_distribute=broadcasted_items,
         )
         return self.__constructor__(new_modin_frame)
+
+    def sort_rows_by_column_values(self, columns, ascending=True, **kwargs):
+        """Reorder the rows based on the lexicographic order of the given columns.
+
+        Parameters
+        ----------
+        columns : scalar or list of scalar
+            The column or columns to sort by
+        ascending : bool
+            Sort in ascending order (True) or descending order (False)
+
+        Returns
+        -------
+        PandasQueryCompiler
+            A new query compiler that contains result of the sort
+        """
+        na_position = kwargs.get("na_position", "last")
```

Comment:
```suggestion
        na_position = kwargs.get("na_position", "last")
        kind = kwargs.get("kind", "quicksort")
```

Reply:
`kind` is not used in this implementation.

Reply:
I see all of your changes now, I will add them.